notebooks/243-tflite-selfie-segmentation/243-tflite-selfie-segmentation.ipynb

```diff
@@ -0,0 +1,707 @@
+{
```

Comment:
<p>Typo in cell title: "Pos<strong>t</strong>process and visualize inference results"</p><p> </p><p>It would be great to have spell check </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>